nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -1,16 +1,16 @@
 # Tests for functions in surf_plotting.py
-
-import tempfile
-
-
 import numpy as np
+import nibabel
+import matplotlib.pyplot as plt
 import pytest
+import tempfile
 
-from matplotlib import pyplot as plt
-
+from nilearn.plotting.img_plotting import MNI152TEMPLATE
 from nilearn.plotting.surf_plotting import (plot_surf, plot_surf_stat_map,
-                                            plot_surf_roi)
+                                            plot_surf_roi, plot_img_on_surf)
+from nilearn.datasets import fetch_surf_fsaverage5
```

Comment:
Not sure how this line was kept in the merge with master, but it needs to be removed !